fuzz/src/utils/test_logger.rs

```diff
@@ -13,19 +13,26 @@ use std::io::Write;
 
 pub trait Output : Clone  + 'static {
 	fn locked_write(&self, data: &[u8]);
+	fn locked_contains(&self, pattern: &str) -> bool;
 }
 
 #[derive(Clone)]
 pub struct DevNull {}
 impl Output for DevNull {
 	fn locked_write(&self, _data: &[u8]) {}
+	fn locked_contains(&self, _pattern: &str) -> bool {
```

Comment:
`DevNull` is actually what's used for honggfuzz (as well as afl/libfuzzer) so this needs support here.

Reply:
Hmmmm isn't the rational of DevNull actually to throw away on the flight the logs, there is not underlying buffer from which to `.contains` ?

Reply:
Yes, but if you're now using the log entries to change the test fail/pass result then you need to change this behavior - currently tests fail in fuzzing but not in `cargo test`. Maybe just use a different logger for `chanmon_consistency`.

Reply:
See https://git.bitcoin.ninja/index.cgi?p=rust-lightning;a=commit;h=8b7ff8b0328feeb788f37645982c249b59579930